### PR TITLE
Upgrade to Ember v2.13

### PR DESCRIPTION
**Pull request by locks**
Builds on the nice work by @bjornharrtell in https://github.com/rust-lang/crates.io/pull/719 and adds some sprinkles.

### CODE REVIEW

**Comment by sgrif** [comment]
Looks like this is failing the build

**Comment by carols10cents** [comment]
yeah @locks there are a few eslint indentation things and then these errors on the ember tests? 

```
not ok 1 PhantomJS 2.0 - Global error: Error: Could not find module `cargo/tests/helpers/register-select-helper` imported from `cargo/tests/test-helper` at http://localhost:7357/assets/vendor.js, line 229

    ---

        Log: |

            { type: 'error',

              text: 'Error: Could not find module `cargo/tests/helpers/register-select-helper` imported from `cargo/tests/test-helper` at http://localhost:7357/assets/vendor.js, line 229\n' }

    ...

not ok 2 PhantomJS 2.0 - Global error: Error: Assertion Failed: The tests file was not loaded. Make sure your tests index.html includes "assets/tests.js". at http://localhost:7357/assets/vendor.js, line 23064

    ---

        Log: |

            { type: 'error',

              text: 'Error: Assertion Failed: The tests file was not loaded. Make sure your tests index.html includes "assets/tests.js". at http://localhost:7357/assets/vendor.js, line 23064\n' }

    ...

```

**Comment by locks** [comment]
Addressing it!

**Comment by locks** [comment]
@carols10cents 👍 ?

File: config/targets.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-env node */
+
+module.exports = {
+  browsers: [
```
Comment (locks) [inline config/targets.js]: @carols10cents make sure this matches https://github.com/rust-lang/rfcs/pull/1985

**Comment by carols10cents** [comment]
Merged this in before it gets too stale, and so we can do this all again next week when the next version of ember is out ;)

Thank you for all your help @bjornharrtell and @locks!!! ❤️

**Pull request merged**